app/assets/v2/css/profile.css

```diff
@@ -146,6 +146,10 @@
   color: #FFCE08;
 }
 
+.profile-section-title {
+  border-bottom: 2px solid #3E24FB;
+}
```

Comment:
Could you reuse the class `.profile-bounties__group-title` and remove this ?

Reply:
make it into a common class and push the unique css into a new class  ? 
I'm not sure 🤔